test/typed/hash_serializer_test.rb

```diff
@@ -1,6 +1,18 @@
 # typed: true
 
 class HashSerializerTest < Minitest::Test
+  class StructWithBooleanDefaultSetToTrue < T::Struct
+    include ActsAsComparable
+
+    const :tired, T::Boolean, default: true
+  end
+
+  class StructWithBooleanDefaultSetToFalse < T::Struct
+    include ActsAsComparable
+
+    const :tired, T::Boolean, default: false
+  end
```

Comment:
I debated just adding something to the job struct but it makes me nervous to keep adding things to that test struct because it's not easy to understand that every piece in there tests a number of things. I decided to just be more explicit and just create structs for specific testcases, this way we don't have one or a few central structs that hold everything together. Thoughts?

Reply:
Yea I think this is fine for testing a specific case.